Source/ASCollectionView.mm

```diff
@@ -207,6 +207,7 @@ @interface ASCollectionView () <ASRangeControllerDataSource, ASRangeControllerDe
     unsigned int collectionViewNumberOfItemsInSection:1;
     unsigned int collectionNodeNodeForItem:1;
     unsigned int collectionNodeNodeBlockForItem:1;
+    unsigned int viewModelForItem:1;
```

Comment:
No prefix needed for this method, since there's no `collectionNode/collectionView` distinction this time.